src/commands/project/deploy/start.ts

```diff
@@ -20,6 +21,7 @@ import { writeConflictTable } from '../../../utils/conflicts';
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/plugin-deploy-retrieve', 'deploy.metadata');
+export const reportsFormatters = Object.keys(DefaultReportOptions);
```

Comment:
probably go to utils/types or something so we don't have commands importing from each other